Changelog.md

```diff
@@ -4,6 +4,9 @@ Expect active development and potentially significant breaking changes in the `0
 
 ### vNext
 
+### 1.4.0
+- Feature: Enhanced typescript definitions to allow for more valid type checking of graphql HOC
```

Comment:
I think this will be a breaking change for TS devs, right? Might be worth a mention in the changelog.